ldap.js

```diff
@@ -83,7 +114,22 @@ exports.getUser = (username, cb) => {
       if (!ret.length) {
         return cb(null, null);
       } else {
-        return cb(null, ret[0]);
+        searchGroups(ret[0].uid, function(err, groups) {
+          if(err) {
+            return cb(null, null);
+          } else {
+            var admin = false;
+            for(var i = 0; i < groups.length; i++) {
```

Comment:
Instead of looping like this:

https://github.com/HelioStrike/openmrs-contrib-atlas/blob/a14678f954026797f5bf9a198fb71b29dbe4fa67/ldap.js#L121-L129

would this work?

```node
// indicate whether or not user is an admin
ret[0].admin = groups.indexOf(ATLAS_ADMIN_LDAP_GROUP) !== -1
```

Reply:
That's an amazing improvement. Thank you!